android/features/music-player/src/main/java/io/newm/feature/musicplayer/service/DownloadManager.kt

```diff
@@ -0,0 +1,28 @@
+package io.newm.feature.musicplayer.service
+
+import android.content.Context
+import android.net.Uri
+import androidx.media3.common.util.UnstableApi
+import androidx.media3.exoplayer.offline.DownloadRequest
+import androidx.media3.exoplayer.offline.DownloadService
+
+interface DownloadManager {
+    fun download(id: String, url: String)
+}
+
+internal class DownloadManagerImpl(
```

Comment:
I pull down the branch and it asked me to make this class public before compiling. Are we able to keep it internal by any chance?